selfservice/hook/web_hook.go

```diff
@@ -525,12 +525,16 @@ func instrumentHTTPClientForEvents(ctx context.Context, httpClient *retryablehtt
 		attempt = retryNumber + 1
 		requestID = uuid.Must(uuid.NewV4())
 		req.Header.Set("Ory-Webhook-Request-ID", requestID.String())
-		reqBody, _ = httputil.DumpRequestOut(req, true)
+		// TODO(@alnr): redact sensitive data
+		// reqBody, _ = httputil.DumpRequestOut(req, true)
+		reqBody = []byte("<redacted>")
 	}
 	httpClient.ResponseLogHook = func(_ retryablehttp.Logger, res *http.Response) {
-		res.Body = io.NopCloser(io.LimitReader(res.Body, 5<<20)) // read at most 5 MB from the response
-		resBody, _ := httputil.DumpResponse(res, true)
-		resBody = resBody[:min(len(resBody), 2<<10)] // truncate response body to 2 kB for event
+		// res.Body = io.NopCloser(io.LimitReader(res.Body, 5<<20)) // read at most 5 MB from the response
+		// resBody, _ := httputil.DumpResponse(res, true)
+		// resBody = resBody[:min(len(resBody), 2<<10)] // truncate response body to 2 kB for event
+		// TODO(@alnr): redact sensitive data
+		resBody := []byte("<redacted>")
```

Comment:
Todo?

Reply:
I haven't figured out how to do this cleanly. That's for another PR.